tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -864,7 +866,9 @@ where
     /// the current span.
     ///
     /// [stored data]: ../registry/struct.SpanRef.html
-    pub fn scope(&self) -> Scope<'_, S>
+    #[deprecated(note = "wraps crate::layer::Context::scope")]
```

Comment:
nit: i think this could explain a little better:
```suggestion
    #[deprecated(note = "wraps layer::Context::scope, which is deprecated")]
```